opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "infer"
+version: "0.1.0"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: "Facebook"
+homepage: "https://github.com/facebook/infer"
+bug-reports: "https://github.com/facebook/infer/issues"
+dev-repo: "https://github.com/facebook/infer.git"
+license: "BSD"
+build: [ [make "-C" "infer" "java"] ]
+install: []
+remove: []
+depends: [
+  "ocamlfind" {build}
+  "sawja" {>="1.5"}
+  "atdgen" {>="1.5"}
+  "javalib" {>="2.3"}
+  "extlib" {>="1.5.4"}
+]
+depexts: [
+  [ ["ubuntu"] ["python2.7-dev"] ]
+  [ ["debian"] ["python2.7-dev"] ]
```

Comment:
Sure -- what homebrew packages are needed to build it on OS X? Python works out of the box, so doesn't need to be specified.



> On 12 Jun 2015, at 14:39, Jeremy Dubreil <notifications@github.com> wrote:
> 
> In opam:
> 
> > +bug-reports: "https://github.com/facebook/infer/issues"
> > +dev-repo: "https://github.com/facebook/infer.git"
> > +license: "BSD"
> > +build: [ [make "-C" "infer" "java"] ]
> > +install: []
> > +remove: []
> > +depends: [
> > +  "ocamlfind" {build}
> > +  "sawja" {>="1.5"}
> > +  "atdgen" {>="1.5"}
> > +  "javalib" {>="2.3"}
> > +  "extlib" {>="1.5.4"}
> > +]
> > +depexts: [
> > +  [ ["ubuntu"] ["python2.7-dev"] ]
> > +  [ ["debian"] ["python2.7-dev"] ]
> Will it be possible to support brew as well with this mechanism ?
> 
> —
> Reply to this email directly or view it on GitHub.
>